dappnode_package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "bitcoin.dnp.dappnode.eth",
+  "name": "bitcoinprune.dnp.dappnode.eth",
```

Comment:
Please, remove the prune references. This is a general bitcoin package, and pruning should be optional.